app/src/main/java/vn/sunasterisk/music_70/MainActivity.kt

```diff
@@ -2,11 +2,14 @@ package vn.sunasterisk.music_70
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import vn.sunasterisk.music_70.base.BaseActivity
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : BaseActivity() {
+
+    override fun getContentViewId(): Int = R.layout.activity_main
+
+    override fun initData(savedInstanceState: Bundle?) {}
```

Comment:
Các hàm blank trong Kotlin nên viết kiểu này.
```suggestion
    override fun initData(savedInstanceState: Bundle?) {
    }
```